dateparser/languages/language.py

```diff
@@ -24,40 +26,51 @@ def __init__(self, shortname, language_info):
             if isinstance(value, int):
                 simplification[key] = str(value)
 
+        self._cached = self
```

Comment:
We can address this isue when we'd have more configuration changes to work with.
What about using `LanguageDataLoader` for now then?